apps/projection/src/app/ui/card/card.component.ts

```diff
@@ -21,14 +26,17 @@ export class CardComponent {
 
```

Comment:
cartType should disappear. 

your component will get less scalable, maintainable everytime you add a new Type.